src/js/claims-status/utils/appeals-v2-helpers.jsx

```diff
@@ -253,64 +253,64 @@ export function getStatusContents(statusType, details = {}, name = {}) {
       contents.title = 'Please review your new Statement of the Case';
       contents.description = (
         <p>The Veterans Benefits Administration sent you a new Statement of the Case on {formattedSocDate}
-        because after completing the remand instructions from the Board, they couldn’t fully grant
+        because, after completing the remand instructions from the Board, they couldn’t fully grant
         your appeal.</p>
       );
       break;
     }
     case STATUS_TYPES.pendingHearingScheduling:
-      contents.title = 'You’re waiting for your hearing to be scheduled';
+      contents.title = 'You’re waiting for the Board to schedule your hearing';
       contents.description = (
-        <p>You requested a {getHearingType(details.type)} hearing on your Form 9. When your hearing is scheduled, you will
-        receive a notice in the mail at least 30 days before the hearing date.</p>
+        <p>You requested a {getHearingType(details.type)} hearing on your Form 9. When the Board schedules your hearing, you’ll
```

Comment:
revert back to

<p>You requested a {getHearingType(details.type)} hearing on your Form 9. When your hearing is scheduled, you will receive a notice in the mail at least 30 days before the hearing date.</p>